test/uk/gov/hmrc/merchandiseinbaggage/pact/MibConnectorContractSpec.scala

```diff
@@ -78,7 +79,7 @@ class MibConnectorContractSpec extends BaseSpecWithApplication with CoreTestData
       interaction
         .description("find a declaration")
         .provided(s"id1234XXX${Json.toJson(declaration.copy(declarationId = DeclarationId("56789"))).toString}")
-        .uponReceiving(GET, s"$declarationsUrl/56789")
+        .uponReceiving(GET, s"${config.mibDeclarationsUrl}/56789")
```

Comment:
Can just be `.uponReceiving(GET, "/declare-commercial-goods/declarations/56789")`